README.md

```diff
@@ -125,6 +125,8 @@ Options:
 
 One can use `omnect-cli` to create a tunneled ssh connection to a device in the field. This is especially useful if the device is behind a NAT and can not directly be contacted. The device must have the `ssh` activated for this. Per default, this command will create a single use ssh key pair, certificate, and ssh configuration to establish a connection to the device.
 
+To create an ssh tunnel, `omnect-cli` must first authenticate against the authentication service. The service credentials vary, depending on whether the device is in the `dev` or `prod` environment.
```

Comment:
Maybe remove chapter or change last sentence to:
The service credentials vary, depending on the omnect cloud environment and default to omnect-prod.